src/util/moneystr.cpp

```diff
@@ -68,7 +68,10 @@ bool ParseMoney(const char* pszIn, CAmount& nRet)
         return false;
     if (nUnits < 0 || nUnits > COIN)
         return false;
-    int64_t nWhole = atoi64(strWhole);
+    int64_t nWhole;
+    if (!ParseInt64(strWhole, &nWhole)) {
```

Comment:
In commit "refactor: Replace use of atoi64 in moneystr.cpp" (2fcdd169cfcb1c0fcfb9d8a31fd9459d39a2dfa8)

I might be missing something looking at this code, but it seems like a possible bug if empty string would have been parse successfully as `0` before but now it will fail to parse. Otherwise behavior seems unchanged since `strWhole` can only contain digit characters.

Reply:
Seems we need a test for ParseMoney that exercises that!